packages/ckeditor5-source-editing/theme/icons/source-editing.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="m12.5 0 5 4.5v15.003h-16V0h11zM3 1.5v2.53l-1.497 1-.003 8 1.5 1v3.974L8 18l-.001 1.504H17.5V8.3L16 9.726l-.004-4.518-4.222-3.692L3 1.5z"/><path d="M4.06 5.89a.75.75 0 0 1 .958 1.149l-.085.071L2.29 9l2.646 1.89a.75.75 0 0 1 .232.951l-.058.095a.75.75 0 0 1-.951.232l-.095-.058-3.5-2.5V8.39l3.496-2.5zm4.194 6.22a.75.75 0 0 1-.958-1.149l.085-.071L10.024 9 7.378 7.11a.75.75 0 0 1-.232-.951l.058-.095a.75.75 0 0 1 .95-.232l.096.058 3.5 2.5v1.22l-3.496 2.5zM10 17.917V20h2.083l6.144-6.144-2.083-2.083L10 17.917zm9.838-5.671a.553.553 0 0 0 0-.784l-1.3-1.3a.553.553 0 0 0-.784 0l-1.016 1.017 2.083 2.083 1.017-1.016z"/></svg>
```

Comment:
![](https://user-images.githubusercontent.com/1099479/120461986-cc021300-c39a-11eb-8032-b2bae2bcedbe.png)

I'd move the code symbol a few px down to improve the inner balance of the icon. Other than this: 👍